src/storage/commands.rs

```diff
@@ -68,6 +68,8 @@ pub enum CommandKind {
     Prewrite {
         /// The set of mutations to apply.
         mutations: Vec<Mutation>,
+        /// Whether each mutation is locked by a pessimistic lock.
```

Comment:
Better to note that this field is only valid for a pessimistic prewrite (for_update_ts > 0)